informer/informer.go

```diff
@@ -98,6 +103,16 @@ func New(config Config) (*Informer, error) {
 	return newInformer, nil
 }
 
+func (i *Informer) Boot(ctx context.Context) error {
+	prometheus.Unregister(prometheus.Collector(i))
```

Comment:
No, there is https://godoc.org/github.com/prometheus/client_golang/prometheus#AlreadyRegisteredError which we can check against on registration and we can just ignore the error then or at best log something.

Reply:
Using it now, this is the way prom checks against this error.